engine/src/test/scala/cromwell/engine/backend/jes/RefreshTokenModeSpec.scala

```diff
@@ -10,7 +10,7 @@ import cromwell.util.EncryptionSpec
 import org.scalatest.{BeforeAndAfterAll, FlatSpec, Matchers}
 import org.specs2.mock.Mockito
 
-class JesBackendSpec extends FlatSpec with Matchers with Mockito with BeforeAndAfterAll with WorkflowDescriptorBuilder {
+class RefreshTokenModeSpec extends FlatSpec with Matchers with Mockito with BeforeAndAfterAll with WorkflowDescriptorBuilder {
   val testWorkflowManagerSystem = new CromwellTestkitSpec.TestWorkflowManagerSystem()
```

Comment:
@kshakir I didn't realize the TestKitSuite was so lightweight and easy to use, thanks. TIL